Sources/SectionScrubber.swift

```diff
@@ -52,6 +45,8 @@ public class SectionScrubber: UIView {
     private let dragGestureRecognizer = UIPanGestureRecognizer()
     private let longPressGestureRecognizer = UILongPressGestureRecognizer()
 
+    private unowned var collectionView: UICollectionView
```

Comment:
I usually use it when I pass the dependency in the initializer. More or less a non-optional weak.